src/models/StationModel.tsx

```diff
@@ -0,0 +1,8 @@
+export interface StationModel {
+    name: string;
+    crs: string;
+}
+
+export const areStationsEqual = ( station1: StationModel, station2: StationModel ) => {
+    return (station1.name == station2.name && station1.crs == station2.crs );
```

Comment:
Inconsistent brackets here, as it's one line we don't need them anyway
```suggestion
    return station1.name == station2.name && station1.crs == station2.crs;
```